.github/workflows/codeql-analysis.yml

```diff
@@ -6,7 +6,16 @@ on:
   pull_request:
     # The branches below must be a subset of the branches above
     branches: [ master ]
-
+    paths-ignore:
+      - 'msvc*/**'
+      - 'check/**'
+      - 'doc/**'
+      - 'packaging/**'
+      - 'snap/**'
+      - '*.md'
+      - 'check/**'
+      - 'default/**'
```

Comment:
CodeQL seems to only apply to C++ files, so I guess this is OK despite eg. https://github.com/freeorion/freeorion/tree/master/default/python

Reply:
Let's keep it simple.  It's not checking python files now. 

Even if we decide to add it to Python, I'd prefer to keep it as a separate task with its own set of paths.